src/scss/components/card/horizontal-card/_horizontal-card-general.scss

```diff
@@ -2,9 +2,29 @@
 
 .su-card-wrapper--horizontal {
   .su-card {
-     // placeholder
+    @apply flex;
+    & > .image {
+      @apply hidden;
+
+      @include grid-media-min('sm') {
+        @apply
+        block
+        mt-0;
```

Comment:
@jenbreese also note that this is our standard format when we are @apply-ing multiple lines in a row.